src/components/map/legend/mock.tsx

```diff
@@ -1,14 +1,4 @@
-import Icon from 'components/icon';
```

Comment:
Only files with JSX syntax have the `.tsx` extension. If there is no component (as in this file), it should be `.ts`.